config/prow/config.yaml

```diff
@@ -666,20 +666,9 @@ tide:
   context_options:
     orgs:
       kubernetes:
-        optional-contexts:
-        - "EasyCLA" # NOTE(cblecker): This is temporary until we fully roll out EasyCLA
```

Comment:
We need to actually swap these out for the old `cla/linuxfoundation` context as that one will not be authoritative. We should leave that in place for at least 1 quarter/release cycle to have things get merged in.

Reply:
Ah, good point. Thanks!